index.d.ts

```diff
@@ -0,0 +1,45 @@
+interface Authentication {
+	username: string;
+	password?: string;
+}
+
+interface Options {
+	width?: number;
+	height?: number;
+	type?: 'png' | 'jpeg';
+	quality?: number;
+	scaleFactor?: number;
+	emulateDevice?: string;
+	fullPage?: boolean;
+	defaultBackground?: boolean;
+	timeout?: number;
+	delay?: number;
+	waitForElement?: string;
+	element?: string;
+	hideElements?: string[];
+	removeElements?: string[];
+	clickElement?: string;
+	modules?: string[];
+	scripts?: string[];
+	styles?: string[];
+	headers?: object;
+	userAgent?: string;
+	cookies?: (string | object)[];
+	authentication?: Authentication;
+	beforeScreenshot?: Function;
+	debug?: boolean;
+	launchOptions?: object;
```

Comment:
This needs a more specific type. Maybe you could import the type from the Puppeteer typings.